sycl/include/sycl/detail/group_sort_impl.hpp

```diff
@@ -361,7 +365,9 @@ convertToOrdered(ValT value) {
 // converts floating type to Ordered (in terms of bitness) type
 template <typename ValT>
 std::enable_if_t<!std::is_same<ValT, OrderedT<ValT>>::value &&
-                     !std::is_integral<ValT>::value,
+                         std::is_floating_point<ValT>::value ||
+                     std::is_same<ValT, sycl::half>::value ||
+                     std::is_same<ValT, sycl::ext::oneapi::bfloat16>::value,
```

Comment:
Shouldn't it be in ( ) ? I.e.:
```suggestion
std::enable_if_t<!std::is_same_v<ValT, OrderedT<ValT>>::value &&
                         (std::is_floating_point_v<ValT> ||
                     std::is_same_v<ValT, sycl::half> ||
                     std::is_same_v<ValT, sycl::ext::oneapi::bfloat16>)
```

Reply:
Fixed. Thank you.